src/LibraryViewExtension/web/library/library.html

```diff
@@ -24,6 +24,10 @@
     <script>
         //Get hold of object registered from C#
         let csharpController = window["controller"];
+        //check if one of our methods does not exist
+        if (csharpController.closeNodeTooltip == null) {
```

Comment:
can you also check `csharpController == null` as well? Windows returns an empty object, and trying to access an property on the empty object will throw exception...

Reply:
LGTM then,.